### ISSUE TITLE
Mermaid diagrams are not rendering when viewed via `file://`

### ISSUE THREAD

**Issue by camold** [squidfunk/mkdocs-material#3560]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

I have tried the configuration of the docs (https://squidfunk.github.io/mkdocs-material/reference/diagrams/#configuration) but the diagram does not render.
The math formulas do render though.

### Expected behaviour

Rendered diagrams.

### Actual behaviour

No rendering of the mermaid diagram. No matter if the external javascript is additionally enabled or not (the docs mention it should not be necessary to add this javascript entry).

![grafik](https://user-images.githubusercontent.com/2109454/152793172-b4dc1863-6425-4dc8-a708-8afec5998960.png)

### Steps to reproduce

```
# my index

Here are the next steps to $b > \frac{1}{n} \sum_{i=1}^n x_i$.

\``` mermaid
graph LR
  A[Start] --> B{Error?};
  B -->|Yes| C[Hmm...];
  C --> D[Debug];
  D --> B;
  B ---->|No| E[Yay!];
\```
Note that the \ are just for correct viewing of the code in markdown.
```

Then run
```
mkdocs build --no-directory-urls
# INFO     -  Cleaning site directory
# INFO     -  Building documentation to directory: /home/camo/Projects/mkdocs_test/site
# INFO     -  Documentation built in 0.40 seconds
```

### Package versions

- Python 3.8.0
- MkDocs: 1.2.3
- Material: 8.1.10+insiders.4.8.1

### Configuration

```yaml
site_name: Site name
site_description: Site description
site_author: Site author

nav:
  - home: index.md

theme:
  name: material

markdown_extensions:
  - pymdownx.superfences:
      custom_fences:
        - name: mermaid
          class: mermaid
          format: !!python/name:pymdownx.superfences.fence_code_format
  - pymdownx.arithmatex:
      generic: true
  - toc:
      permalink: "¤"

extra_javascript:
  # - https://unpkg.com/mermaid/dist/mermaid.min.js

  - javascripts/mathjax.js
  - https://polyfill.io/v3/polyfill.min.js?features=es6
  - https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js
```

### System information

- Operating system: Linux 5.4.0-87-generic #98~18.04.1-Ubuntu SMP Wed Sep 22 10:45:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

- Browser: Firefox 93.0 (64-bit)

**Comment by camold** [comment]
This is what I see:
![grafik](https://user-images.githubusercontent.com/2109454/152929412-82327fe9-3c5f-4d91-a8a5-736884fa1167.png)

Thanks for considering so fast.

**Comment by squidfunk** [comment]
Thanks. That's not related to Material for MkDocs. Sounds like misconfiguration then. Are you sure you are using Insiders? Please check the `generator` meta tag in the browser. It's located in the head and should print:

<img width="578" alt="Bildschirmfoto 2022-02-08 um 09 13 55" src="https://user-images.githubusercontent.com/932156/152945360-97590314-aa1a-4a42-a23f-cd6dfd16bdd8.png">

**Comment by squidfunk** [comment]
I tried reproducing your issue with the exact configuration you provided, to no avail:

<img width="695" alt="Bildschirmfoto 2022-02-08 um 09 32 10" src="https://user-images.githubusercontent.com/932156/152948318-14387cb1-93ca-4657-b768-11a25104b1ac.png">

**Comment by camold** [comment]
I've attached the rendered static website. Maybe that helps so you can have a deep look yourself. Hope the upload doesn't violate any github rules.
[site.zip](https://github.com/squidfunk/mkdocs-material/files/8024419/site.zip)

**Comment by squidfunk** [comment]
Thanks. I've found the problem:

```
Fetch API cannot load file:///Users/squidfunk/Downloads/site%202/assets/stylesheets/vendor/mermaid.53552a32.min.css. URL scheme "file" is not supported.
```

>ou're viewing it locally and the current integration doesn't work on `file://`. I'll see if and how we can fix that.

**Comment by squidfunk** [comment]
Released as part of 8.1.11-insiders-4.8.3.

**Comment by camold** [comment]
Just getting back to this now. Will check as soon as possible. Thanks for the quick fix!
